Aspire Budgeting/Facilities/AppDefaults.swift

```diff
@@ -10,12 +10,14 @@ protocol AppDefaults {
   func getDefaultFile() -> File?
   func addDefault(file: File)
   func addDataMap(map: [String: String])
+  func getDataMap() -> [String: String]
 }
 
 protocol AppUserDefaults {
   func data(forKey defaultName: String) -> Data?
   func set(_ value: Any?, forKey defaultName: String)
   func removeObject(forKey defaultName: String)
+  func dictionary(forKey defaultName: String) -> [String : Any]?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/d5353e560168880237460fc75a2a6ee44e50449b/Aspire Budgeting/Facilities/AppDefaults.swift#L20"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>AppDefaults.swift:20</code></td>
    </tr>
  </tbody>
</table>